src/expertsystem/reaction/conservation_rules.py

```diff
@@ -4,13 +4,12 @@
 quantum numbers of the reaction.
 
 A rule is a function that takes quantum numbers as input and outputs a boolean.
-There are four different types of rules:
+There are three different types of rules:
 
-1. Rules that work on individual graph edges (WIP).
-2. Rules that work on individual graph nodes (WIP).
-3. `EdgeQNConservationRule` that work on the interaction level, which use
+1. `GraphElementRule` that work on individual graph edges or nodes.
+2. `EdgeQNConservationRule` that work on the interaction level, which use
```

Comment:
Perhaps for the 'symmetry' between nodes and edges we need to think of naming here: `EdgeConservationRule` and `NodeConservationRule`. But not sure if I understand the intention correctly.

Reply:
See https://github.com/ComPWA/expertsystem/pull/368#discussion_r517947995